tutorials/llama_index_knowledge_base_tutorial.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    os.environ["OPENAI_API_KEY"] = "copy paste your api key here"
</pre><p>may want to use <a href="https://docs.python.org/3/library/getpass.html" target="_blank">getpass</a> instead so it doesn't get saved along with the notebook</p><p><code>os.environ["OPENAI_API_KEY"] = getpass.getpass(prompt="paste your api key here")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>